smoketest/make-bindings.sh

```diff
@@ -23,5 +23,6 @@ fi
 # Fetch metadata from BridgeHub and generate client
 subxt codegen --url ws://localhost:11144 > src/parachains/bridgehub.rs
 subxt codegen --url ws://localhost:12144 > src/parachains/assethub.rs
-subxt codegen --url ws://localhost:9944  > src/parachains/relaychain.rs
 subxt codegen --url ws://localhost:13144 > src/parachains/template.rs
```

Comment:
Is template chain still in use or just safe to remove?

Reply:
Not safe yet, we will still need to add an extrinsic to penpal to call create_agent/create_channel. Although penpal has `sudo` pallet so we actually dont need to add anything and can just use a sudo call with pallet_xcm.send to call bridgehub.